source.go

```diff
@@ -100,6 +100,15 @@ type SourceOwner struct {
 	VerifiedPhone   string   `json:"verified_phone"`
 }
 
+// RedirectFlowFailureReason represents the possible failure reasons of a redirect flow.
+type RedirectFlowFailureReason string
+
+const (
+	RedirectFlowFailureReasonUserAbort       RedirectFlowFailureReason = "user_abort"
+	RedirectFlowFailureReasonDeclined        RedirectFlowFailureReason = "declined"
+	RedirectFlowFailureReasonProcessingError RedirectFlowFailureReason = "processing_error"
+)
```

Comment:
Minor OB, but we recently went through and alphabetized everything in the library that we could find. Unfortunately there's no CI check on this, but would you mind giving this list of consts the same treatment? Thanks!

Reply:
👍